fe1-web/src/features/wallet/components/RollCallTokensDropDown.tsx

```diff
@@ -0,0 +1,50 @@
+import { Picker } from '@react-native-picker/picker';
+import PropTypes from 'prop-types';
+import React, { useMemo } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { RollCallToken } from '../objects/RollCallToken';
+
+const styles = StyleSheet.create({
+  pickerStyle: {
+    maxHeight: 40,
+    width: 'fit-content',
+    maxWidth: '90%',
+    fontSize: 20,
+    textAlign: 'center',
+    margin: 'auto',
+  },
+});
+const RollCallTokensDropDown = (props: IPropTypes) => {
+  const { onTokenChange } = props;
+  const { rollCallTokens } = props;
+
+  const options = useMemo(() => {
+    return rollCallTokens.map((rc) => {
+      const value = rc?.token.publicKey.valueOf() || '';
+      return <Picker.Item key={value} label={value} />;
+    });
+  }, [rollCallTokens]);
```

Comment:
Maybe that is just me but as long as there are no performance problems with this code (are there any?), I find it confusing to have some UI components memoized instead of just having this code inline on line 38?

Reply:
True, I thought about the lot of tokens case but even then it might not be an issue